### PR TITLE
KOGITO-1564 Fix Travel agency issues on Pom, causing errors on native build

### PR DESCRIPTION
**Pull request by tiagodolphine**
Fixing some issues with native build on travel agency.
Fixing scripts issues on travel agency.

### CODE REVIEW

**Comment by ricardozanini** [comment]
Relates: https://github.com/kiegroup/kogito-examples/pull/185

**Review by cristianonicolai — APPROVED** [review]

**Comment by evacchi** [comment]
does this close https://issues.redhat.com/browse/KOGITO-1641 ?

**Review by evacchi — APPROVED** [review]

**Comment by evacchi** [comment]
please add the KOGITO JIRA ticket to the title

**Comment by tiagodolphine** [comment]
@evacchi sure, Done I added the same as the runtime because I used the travel agency to validate it.

File: .gitignore [hunk 1]
```diff
@@ -42,3 +41,8 @@ test.properties
 *.tlog
 
 workspace
+
+#Scripts
+data-index-service*.jar
```
Comment (radtriste) [inline .gitignore:46]: is that needed ?
can't we just ignore all target folders ?

Comment (tiagodolphine) [reply .gitignore:46]: @radtriste the jars are not on target folders, the script downloads the jars, it is not building data-index/management console.

Comment (radtriste) [reply .gitignore:46]: oki

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
cherry-pick in 0.9.x handled in https://github.com/kiegroup/kogito-examples/pull/188

**Pull request merged**